doc/playbook.rst

```diff
@@ -70,43 +71,10 @@ Any subsequent builds of the service container will build against both bases.
 
 This is the only situation in which a change to ``BASE_TAG_LIST`` should be committed to ``main``.
 
-Building the Science Pipelines Manually (aka "Quick-Stack Build")
------------------------------------------------------------------
-It will sometimes be necessary to compile a container with the LSST Science Pipelines manually.
+It will sometimes be necessary to compile a container with the LSST Science Pipelines manually (called a Quick-Stack build).
```

Comment:
This commit largely reverses "Add directions to playbook for Science Pipelines quick-builds ". Would it be possible to merge the two so that the commit history looks more coherent?

Reply:
I meant merge _only_ those two commits; now it's impossible to [tell the formatting changes from the content ones](https://developer.lsst.io/work/flow.html#git-commit-organization-best-practices).